src/otx/cli/install.py

```diff
@@ -132,6 +135,16 @@ def otx_install(option: str | None = None, verbose: bool = False, do_not_install
             status_code = mim_installation(mmcv_install_args)
             if status_code == 0:
                 console.log(f"MMLab Installation Complete: {mmcv_install_args}")
+            else:
+                msg = "Cannot complete installation"
+                raise RuntimeError(msg)
+
+        # TODO(harimkang): Remove this reinstalling after resolving conflict with anomalib==1.0.1
+        # https://github.com/openvinotoolkit/training_extensions/actions/runs/8531851027/job/23372146228?pr=3258#step:5:2587
+        status_code = create_command("install").main(["jsonargparse==4.27.7"])
+        if status_code != 0:
+            msg = "Cannot install jsonargparse==4.27.7"
+            raise RuntimeError(msg)
```

Comment:
@ashwinvaidya17 Can't we keep `jsonargparse` up to date in `anomalib`, it seems to be a fixed version now. So it seems to crash now when we try to use the latest version.

Reply:
Ah i found latest version of `anomalib` (develop) use `jsonargparse==4.27.7` https://github.com/openvinotoolkit/anomalib/blob/51e190adb731572826c91789435a992146a39bc2/pyproject.toml#L21
@ashwinvaidya17 @samet-akcay But in `1.0.1`, it's still fixed to the old version and it raise conflict issue. (+ I think `timm` is in a similar situation now.)

Reply:
Since there has been no response for a long time, I marked this as resolved.